microsoft-edge/devtools-guide-chromium/memory-problems/heap-snapshot-schema.md

```diff
@@ -93,7 +93,7 @@ The `.heapsnapshot` JSON data contains a root object that has the following prop
 | `trace_function_infos` | _Not documented yet_ | `Array` |
 | `trace_tree` | _Not documented yet_ | `Array` |
 | `samples` | _Not documented yet_ | `Array` |
-| `locations` | _Not documented yet_ | `Array` |
+| `locations` | Contains information about the location of nodes. To parse this data, use `snapshot.meta.location_fields` with the `nodes` array. | `Array` |
```

Comment:
```suggestion
| `locations` | Contains information about the script location of nodes. To parse this data, use `snapshot.meta.location_fields` with the `nodes` array. | `Array` |
```

The word "location" could mean different things in different contexts. If I'm understanding this right, it's where a node was created in JavaScript, right? If so, then my suggestion might help.